### PR TITLE
Updating javascript dependancies on dated reminders

### PR DESCRIPTION
**Pull request by Gicheha**

### CODE REVIEW

File: sites/default/sqlconf.php [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
 $login	= 'libreehr';
 $pass	= 'libreehr';
 $dbase	= 'libreehr';
 
+global $disable_utf8_flag;
+$disable_utf8_flag = false;
+
 $sqlconf = array();
 global $sqlconf;
 $sqlconf["host"]= $host;
```
Comment (muarachmann) [inline sites/default/sqlconf.php]: @Gicheha please add this file to your gitignore file. this is not suppose to be part of this commit.I think we corrected this last time

File: library/js/jquery-3.4.0.min.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.4.0 | (c) JS Foundation and other contributors | jquery.org/license */
```
Comment (muarachmann) [inline library/js/jquery-3.4.0.min.js]: This is not needed now in the codbase. The standard version should be `3.3.1` as you called in `call_required_libraries(...`

**Comment by Gicheha** [comment]
@muarachmann I thought with the latest code you had it on gitignore...I am working on it

**Comment by muarachmann** [comment]
Either you haven't pulled the latest codes or you haven't debased your
branch. Just avoid the sqlconfig file in your commits henceforth

**Comment by Gicheha** [comment]
@muarachmann...I have made the necessary changes

File: library/js/jquery-3.4.0.min.js (hunk 2 above)
Comment (muarachmann) [reply library/js/jquery-3.4.0.min.js]: Drop(delete) this.

File: library/js/jquery-3.3.1.min.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
...e(n,n),n}},w.each(w.expr.match.bool.source.match(/\w+/g),function(e,t){var n=ht[t]||w.find.attr;ht[t]=function(e,t,r){var i,o,a=t.toLowerCase();return r||(o=ht[a],ht[a]=i,i=null!=n(e,t,r)?a:null,ht[a]=o),i}});var gt=/^(?:input|select|textarea|button)$/i,yt=/^(?:a|area)$/i;w.fn.extend({prop:function(e,t){return z(this,w.prop,e,t,arguments.length>1)},removeProp:function(e){return this.each(function(){delete this[w.propFix[e]||e]})}}),w.extend({prop:function(e,t,n){var r,i,o=e.nodeType;if(3!==o&&8!==o&&2!==o)return 1===o&&w.isXMLDoc(e)||(t=w.propFix[t]||t,i=w.propHooks[t]),void 0!==n?i&&"set"in i&&void 0!==(r=i.set(e,n,t))?r:e[t]=n:i&&"get"in i&&null!==(r=i.get(e,t))?r:e[t]},propHooks:{tabIndex:{get:function(e){var t=w.find.attr(e,"tabindex");return t?parseInt(t,10):gt.test(e.nodeName)||yt.test(e.nodeName)&&e.href?0:-1}}},propFix:{"for":"htmlFor","class":"className"}}),h.optSelected||(w.propHooks.selected={get:function(e){var t=e.parentNode;return t&&t.parentNode&&t.parentNode.selectedIndex,null},set:function(e){var t=e.parentNode;t&&(t.selectedIndex,t.parentNode&&t.parentNode.selectedIndex)}}),w.each(["tabIndex","readOnly","maxLength","cellSpacing","cellPadding","rowSpan","colSpan","
```
Comment (muarachmann) [inline library/js/jquery-3.3.1.min.js]: this is already called in `call_required libraries()` which is gotten from the `asset/` folder. No need including it in the `library/js....`. Please remove this as well. I think this PR needs just one file edited and that is  `interface/main/dated_reminders/dated_reminders_add.php` which seems perfect but these other files are :0

**Pull request closed without merging**